e2e/__tests__/cliHandlesExactFilenames.test.ts

```diff
@@ -30,7 +27,9 @@ test('CLI accepts exact file names if matchers matched', () => {
 
   expect(result.exitCode).toBe(0);
 
-  const {rest, summary} = extractSummary(result.stderr);
+  const {rest, summary} = extractSummary(
+    result.stderr.replace('\\\\foo\\\\bar', '\\/foo\\/bar'),
```

Comment:
Could we normalize the output of jest instead of normalizing in the test itself?

Reply:
Technically, yes but I don't know how much work that'll involve.

However, I don't know if thats a good idea from UX POV, as jest is implicitly normalizing the input - we pass in `./foo/bar.spec.js`, which is a Linux style file path.

By outputting the actual OS-specific file path that was used, users can see that it doesn't matter if they pass in Linux file paths; where as if we normalised them in the output here, personally I'd look at that and think "oh maybe Jest isn't finding my test file because it's using the Linux file path" (especially since copying & pasting the file path into Windows would cause an error, as iirc Powershell, CMD, and Explorer all expect forward slashes).